docs/data-sources/service_level_objective.md

```diff
@@ -38,6 +38,7 @@ data "datadog_service_level_objective" "api_slo" {
 - `description` (String) The description of the service level objective.
 - `name` (String) Name of the Datadog service level objective
 - `query` (List of Object) The metric query of good / total events (see [below for nested schema](#nestedatt--query))
+- `tags` (Set of String) List of tags associated with the service level objective.
```

Comment:
```suggestion
- `tags` (Set of Strings) List of tags associated with the service level objective.
```

Reply:
The readme is generated by [terraform-plugin-docs](https://github.com/hashicorp/terraform-plugin-docs)
I ran the command `make docs`

Reply:
Indeed. This is expected and consistent with our docs overall.